src/shogun/preprocessor/RandomFourierGaussPreproc.h

```diff
@@ -32,9 +32,8 @@ namespace shogun {
 	 * 1177-1184).
 	 *
 	 */
-	class RandomFourierGaussPreproc
-	    : public RandomMixin<DensePreprocessor<float64_t>>
-	{
+class RandomFourierGaussPreproc: public RandomMixin<DensePreprocessor<float64_t>>
```

Comment:
yes this is an issue that class_list has. You could open an issue to explains how to reproduce the problem, and then someone can fix it one day

Reply:
I could, but the problem is, as long as that isn't fixed this PR won't be viable for merging. I don't think it'd be a good idea to wait for some ideal person to fix this while this code rots.

Reply:
Documenting this is definitely a good idea.
You can just put it all in one line and it will work as a workaround here.

Reply:
I'm going with the workaround for now. I'll go with your advice on opening an issue regarding this.